### DISCUSSION TITLE
FullStack .NET - Graficos de Linhas

### DISCUSSION THREAD

**Discussion by rogercomp** [balta-io/forum#673] [Tira Dúvidas] [answered]
O grafico de linhas esta com os dados do eixo y cortados a esquerda

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @rogercomp, como vai?
Poderia me dar mais detalhes e um print?

**Reply by rogercomp** [reply]
<img width="209" alt="ss" src="https://github.com/user-attachments/assets/0e13411c-41bc-4310-baa7-091815713fed">

**Reply by andrebaltieri** [reply]
Para mim, parece OK!

**Comment by rogercomp** [comment]
![image](https://github.com/user-attachments/assets/53073e21-4542-4473-8de1-5c8a6e357f4d)
Esse e o video que vc estava gravando incluindo até os minutos do video (Grafico/Resumo Financeiro)

**Reply by rogercomp** [reply]
![image](https://github.com/user-attachments/assets/a15695cc-4f6e-4a01-b2fd-90d006287979)

**Reply by andrebaltieri** [reply]
Bom dia, @rogercomp, não entendi sua questão ainda... pode sinalizar nas imagens o que está errado e como seria o correto?

**Comment by rogercomp** [comment]
Bom dia Balta, acho que vc nao olhou no detalhe, perceba que vc redimensionando a tela os numeros são cortados(amplia a imagem q eu mandei pra vc ver, faz esse teste ai tbm para comprovar

**Reply by andrebaltieri** [reply]
Não consigo ver, a qualidade da imagens está muito ruim!
